tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "module":"commonjs",
     "sourceMap": true,
     "allowJs": true,
-    "checkJs": true,
+    "checkJs": false,
```

Comment:
Rather than turn on the stricter `-checkJs` switch, I like the idea of renaming the source files to `.ts`, on a separate branch of course.